dist/index.html

```diff
@@ -0,0 +1,48 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Todo Project</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    <h1>Your To-Do App</h1>
+    <div class="content">
+        <div class="form-container">
+            <button class="btn" id="add-todo-button">Add New ToDo</button>
+            <div id="add-todo-form" style="display:none">
+                <form id="add-todo">
+                    <label for="Title">Title:</label>
+                    <input type="text" id="Title" name="Title" placeholder="Title" required>
```

Comment:
id name should not start with capital latter, please change from `Title` => `title`

also update on other place as well